lib/router/src/utils.ts

```diff
@@ -6,8 +6,13 @@ interface StoryData {
   storyId?: string;
 }
 
+interface SeparatorOptions {
+  rootSeparator: string | RegExp;
+  groupSeparator: string | RegExp;
+}
+
 export const knownNonViewModesRegex = /(settings)/;
-const splitPath = /\/([^/]+)\/([^/]+)?/;
+const splitPathRegex = /\/([^/]+)\/([^/]+)?/;
```

Comment:
Can we change this to something else given it has nothing to do with the `splitPath` function?

Reply:
@tmeasday Renamed the functions instead. LMK what you think.

Reply:
WFM :+1: